.webpack/webpack.dev.js

```diff
@@ -9,10 +9,12 @@ const path = require('path');
 const webpack = require('webpack');
 const { merge } = require('webpack-merge');
 
-const common = require('./webpack.common');
+const base = require('./webpack.base');
+
 const projectRootDir = path.resolve(__dirname, '..');
 
-module.exports = merge(common, {
+module.exports = merge(base('development'), {
+  devtool: 'inline-source-map',
```

Comment:
We usually use `eval-source-map`. Which `devtool` is preferable and what are the advantages and disadvantages?

Reply:
https://webpack.js.org/configuration/devtool/


I picked what I use in development and what works best. Want me to use eval-source-map?

Reply:
Hmm.. It looks like `eval-source-map` [is not supported by TerserWebpackPlugin](https://webpack.js.org/plugins/terser-webpack-plugin/#note-about-source-maps).

We don't have a preference either way, so long as it works and we are able to hit breakpoints. I was just curious if there was some other reason to switch to `inline-source-map`.

Reply:
I've used inline-source-map for a lot of applications, and it has a nice stacktrace and filesize that doesn't make the DX of webpack worse